### PR TITLE
Deduct Deposit from Order Summary

### PR DESCRIPTION
**Pull request by sbassah**
Original Gist: https://gist.github.com/spivurno/da655f24c07e6204a11e2e647113677f

### CODE REVIEW

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (spivurno) [inline gp-ecommerce-fields/gpecf-deduct-deposit.php:6]: Can you remove the extra space before the plugin name?

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (spivurno) [inline gp-ecommerce-fields/gpecf-deduct-deposit.php:22]: Update this comment to read:
> // Update "123" to the ID of your form.

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (spivurno) [inline gp-ecommerce-fields/gpecf-deduct-deposit.php:27]: Update this comment to read:
> // Update the "2" to your deposit field ID.

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (spivurno) [inline gp-ecommerce-fields/gpecf-deduct-deposit.php:5]: Let's add a brief description here and mention that there are changes that must be made inline.

**Review by spivurno — CHANGES_REQUESTED** [review]
A few minor changes but I'd like to take another look at this one before we merge. 🙏

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (sbassah) [reply gp-ecommerce-fields/gpecf-deduct-deposit.php]: Done, but I was wondering if this should be 4 instead of 2 per the SL Code organization formating documentation.

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (sbassah) [reply gp-ecommerce-fields/gpecf-deduct-deposit.php]: Done

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (sbassah) [reply gp-ecommerce-fields/gpecf-deduct-deposit.php]: Done

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (sbassah) [reply gp-ecommerce-fields/gpecf-deduct-deposit.php:5]: Done

File: gp-ecommerce-fields/gpecf-deduct-deposit.php
Comment (spivurno) [reply gp-ecommerce-fields/gpecf-deduct-deposit.php]: Great catch! Yes, let's go ahead and update that.

**Review by spivurno — APPROVED** [review]
Approved pending the change you suggested.

**Pull request merged**
